lib/gitpod.ts

```diff
@@ -1,85 +1,34 @@
-import cdk = require('@aws-cdk/core');
+import {
+    Construct,
+    Stack,
+    StackProps,
+    Duration
+} from '@aws-cdk/core';
+import { NodejsFunction } from '@aws-cdk/aws-lambda-nodejs';
+
 import { KubernetesManifest } from '@aws-cdk/aws-eks';
 
 import { readYamlDocument, loadYaml } from './charts/utils';
-import { Database } from './database';
-import { Registry } from './registry';
+//import { Database } from './database';
+//import { Registry } from './registry';
 import { importCluster } from './charts/cluster-utils';
 
-var createNestedObject = function (base: any, names: any, value: any) {
-    var lastName = arguments.length === 3 ? names.pop() : false;
-    for (var i = 0; i < names.length; i++) {
-        base = base[names[i]] = base[names[i]] || {};
-    }
-
-    if (lastName) {
-        base = base[lastName] = value;
-    }
-};
-
-// TODO: switch to official gitpod.io build.
-const version = "aledbf-mk3.68";
-
-export interface GitpodProps extends cdk.StackProps {
+export interface GitpodProps extends StackProps {
     domain: string
 
     certificateArn?: string
 
-    database: Database
-    registry: Registry
+    //database: Database
```

Comment:
Are these just dead code now or are we leaving a todo for future us?

Reply:
removed